lib/src/conflicts.rs

```diff
@@ -250,6 +225,78 @@ pub enum ConflictMarkerStyle {
     Git,
 }
 
+/// Represents a kind of conflict marker which can be materialized and parsed.
+#[derive(Clone, Copy, PartialEq, Eq)]
+#[repr(u8)]
+enum ConflictMarkerKind {
```

Comment:
I'm not sure how I feel about "separator". I personally have always thought of each individual line like "<<<<<<<" as being markers, rather than just being a part of a marker. Maybe `ConflictMarkerLine` and `ConflictMarkerLineChar` would be clearer? I don't feel strongly though.

Reply:
I think `ConflictMarkerChar`, `LineChar`, `LineKind` are good, if the original concern was that `ConflictMarkerKind` sounds like a `ConflictMarkerStyle`.